pyaedt/edb_core/siwave.py

```diff
@@ -109,7 +111,7 @@ def do_pin_grouping(self,value):
 
 class CircuitPort(Source):
     def __init(self):
-        super().__init__()
+        Source.__init__(self)
```

Comment:
```suggestion
        super(CircuitPort, self).__init__()
```
This is compatible with Python 2.x and more indicated if Source class inherits from ``object`` like  ``class Source(object):``.
Use ``Source.__init__(self)`` only when Source is defined as classic class ``class Source():``